src/Markdig/Renderers/TextRendererBase.cs

```diff
@@ -90,7 +90,7 @@ protected TextRendererBase(TextWriter writer) : base(writer)
             indents = new List<string>();
         }
 
-        internal void Reset()
```

Comment:
Wondering if we shouldn't rename this method to `ResetInternal` that would call a `protected virtual void Reset()` method instead?
@MihaZupan ?

Reply:
I don't think we need the internal method in that case.

`protected internal virtual void Reset()`?

I've never seen that signature, but apparently it's legal.

Reply:
Though I'm not sure what we'd be getting from making it virtual - we would never call it on renderers we don't own (since the writer might not support it)

Reply:
From what I can see, the `Reset` method is always invoked by the code having control over `HtmlRenderer` instantiation.
So there is no point in making the method virtual - basically what @MihaZupan said in the previous comment.

Custom `HtmlRenderer` inheritors will be able to hide the method with a virtual one, if they need virtual behavior for their derived types.